.github/workflows/build-documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: build-documentation
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
```

Comment:
Could you please explain advantages of this action in comparison with plain `apt install -y python`?
I have no objections against the actions if it works fast enough (maybe even faster than `apt install`.

Reply:
Not only python, we also need to install python-pip. The "setup-python@v2" action already has python and pip.